background.js

```diff
@@ -46,83 +45,88 @@ chrome.runtime.onMessage.addListener(
 			default:
 				console.log(request);
 		}
-	});
-	
-if(initStorage("v0.8.92", true)) {
-	// v0.7
-	initStorage("popupcolor", "blue");
-	initStorage("highlight", true);
-	
-	// v0.8
-	// No changes to options
-	
-	// V0.8.5
-	initStorage("textboxhl", false);
+	})
+var getItems = ["popupcolor", "highlight", "textboxhl", "onlyreading",

```

Comment:
Two things:

1) We should share this array of option with options.js using a global object like `optionNames`
2) We should probably make it easier to read by initializing it with one option name per line in alphabetical order.

Reply:
done.

> Two things:
> 
> 1. We should share this array of option with options.js using a global object like `optionNames`
> 2. We should probably make it easier to read by initializing it with one option name per line in alphabetical order.

Done.